pkg/sdkserver/types.go

```diff
@@ -58,7 +58,7 @@ type toolOrFileRequest struct {
 	ChatState            string   `json:"chatState"`
 	Workspace            string   `json:"workspace"`
 	Env                  []string `json:"env"`
-	CredentialContext    string   `json:"credentialContext"`
+	CredentialContext    []string `json:"credentialContext"`
```

Comment:
Darren specifically asked me to change this in the API. My SDK changes haven't been merged yet so this isn't really a breaking change, since nothing should be using it.

Reply:
It's a breaking change because older versions of the SDK won't work with the newer versions.

If Darren specifically asked for it, then that's fine.

Reply:
ohhhh I just noticed that this is in the "toolOrFileRequest" and not in the Credentials struct. My bad. You are correct about this